cnames_active.js

```diff
@@ -3376,6 +3376,7 @@ var cnames_active = {
   "web.quizlet": "redyetidev.github.io/Quizlet.JS-Web", // noCF
   "web230": "paylesworth.github.io/web230",
   "webassembly": "xtuc.github.io/webassemblyjs",
+  "webconnect":"nuzulul.github.io/webConnect.js",
```

Comment:
```suggestion
  "webconnect": "nuzulul.github.io/webConnect.js",
```
As the CI error notes, you're missing a space